src/components/event-log/event-log.stories.mdx

```diff
@@ -0,0 +1,32 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+import events from './demo/events.json';
+import eventsDemo from './demo/events.twig';
+const eventsDemoStory = (args) => eventsDemo({ items: events, ...args });
+
+<Meta title="Components/Event Log" />
+
+# Event Log
+
+Displays a list of event details in chronological order. Designed to show the presentation history for talks, workshops or other events we’ve conducted.
+
+For a less specialized display of tabular data, consider using [Table](/?path=/docs/components-table--basic). For simple string/number pairs, consider [Dot Leader](/?path=/docs/components-dot-leader--list). To display a single or upcoming date, consider using [Calendar Date](/?path=/docs/components-calendar-date--basic) (optionally combined with [Media](/?path=/docs/objects-media--event-summary)).
```

Comment:
This is great! :+1:

I'd like to see more of this "when not to use this component" type documentation in component libraries